{{ cookiecutter.package_name|slugify }}/README.md

```diff
@@ -56,37 +62,29 @@ To add and install this package as a dependency of your project, run `poetry add
     EOF
     ```
 {%- if cookiecutter.private_package_repository_name %}
-1. [Configure Poetry's credentials for this package's private repository](https://python-poetry.org/docs/repositories/#configuring-credentials):
+1. [Add your private package repository credentials to your Poetry's `auth.toml` file](https://python-poetry.org/docs/repositories/#configuring-credentials):
+    ```toml
+    # Linux:   ~/.config/pypoetry/auth.toml
+    # macOS:   ~/Library/Application Support/pypoetry/auth.toml
+    # Windows: C:\Users\%USERNAME%\AppData\Roaming\pypoetry\auth.toml
+    [http-basic.{{ cookiecutter.private_package_repository_name|slugify }}]
+    username = "{username}"
+    password = "{password}"
+    ```
+1. Create a `.env` file in the project directory that [Docker Compose reads](https://docs.docker.com/compose/env-file/) to pass Poetry's `auth.toml` file as a [build and run time secret](https://docs.docker.com/compose/compose-file/compose-file-v3/#secrets-configuration-reference):
     ```sh
-    # bash
-    echo "export POETRY_HTTP_BASIC_{{ cookiecutter.private_package_repository_name|upper|replace("-", "_") }}_USERNAME='{username}'" >> ~/.bash_profile
-    echo "export POETRY_HTTP_BASIC_{{ cookiecutter.private_package_repository_name|upper|replace("-", "_") }}_PASSWORD='{password}'" >> ~/.bash_profile
-    
-    # fish
-    echo "set --export POETRY_HTTP_BASIC_{{ cookiecutter.private_package_repository_name|upper|replace("-", "_") }}_USERNAME '{username}'" >> ~/.config/fish/config.fish
-    echo "set --export POETRY_HTTP_BASIC_{{ cookiecutter.private_package_repository_name|upper|replace("-", "_") }}_PASSWORD '{password}'" >> ~/.config/fish/config.fish
-
-    # zsh
-    echo "export POETRY_HTTP_BASIC_{{ cookiecutter.private_package_repository_name|upper|replace("-", "_") }}_USERNAME='{username}'" >> ~/.zshenv
-    echo "export POETRY_HTTP_BASIC_{{ cookiecutter.private_package_repository_name|upper|replace("-", "_") }}_PASSWORD='{password}'" >> ~/.zshenv
+    # Linux
```

Comment:
The Cookiecutter now assumes that we have Poetry installed on our machines as well. Well, kind of, it's just a config file after all, but it is in a very specific location! I suppose we should make that explicit?

Reply:
Indeed, it's not required to have Poetry installed on the host system, but the way of storing the credentials is indeed Poetry-specific. By storing the credentials in that location, we have the benefit that if you _do_ happen to use Poetry on the host system, that authentication will work out of the box on the host as well.

What would you like to make more explicit exactly?